integration-tests/runtime-tests/test/tests/mosaic/mosaicTests.ts

```diff
@@ -0,0 +1,419 @@
+import { expect } from 'chai';
+import {ApiPromise, Keyring} from "@polkadot/api";
+import testConfiguration from './test_configuration.json';
+import { sendAndWaitForSuccess } from '@composable/utils/polkadotjs';
+import {KeyringPair} from "@polkadot/keyring/types";
+import { mintAssetsToWallet } from '@composable/utils/mintingHelper';
+import { send } from 'process';
+import {
+    CommonMosaicRemoteAssetId,
+    OrmlTokensAccountData,
+    PalletMosaicDecayBudgetPenaltyDecayer,
+    PalletMosaicNetworkInfo
+} from "@composable/types/interfaces";
+import {start} from "repl";
+import {AccountId} from "@polkadot/types/interfaces/runtime";
+import { doesNotMatch } from 'assert';
```

Comment:
A lot of unused imports, can be reduced to:
```suggestion
import { expect } from 'chai';
import testConfiguration from './test_configuration.json';
import { sendAndWaitForSuccess } from '@composable/utils/polkadotjs';
import {KeyringPair} from "@polkadot/keyring/types";
import { mintAssetsToWallet } from '@composable/utils/mintingHelper';
import {
    CommonMosaicRemoteAssetId,
    OrmlTokensAccountData,
    PalletMosaicDecayBudgetPenaltyDecayer,
    PalletMosaicNetworkInfo
} from "@composable/types/interfaces";
```